app.js

```diff
@@ -38,12 +42,25 @@ function clientLoaded (client) {
   });
 }
 
+/** Utility function for seeing if an error is fatal (crashes the program)
+ * @param {Object} err - the error in question.
+ * @return {boolean} - if the error is fatal or not
+ */
+function isFatal(err) {
+  if (err.name === 'EarlyHangup' || err.anme === 'HangupFailure' ||
+      err.name === 'NoStations') {
+        return true;
+      } else {
+        return false;
+      }
+}
+
 /**
  * Handles errors found in application.
  * @param {Object} err - error from application.
  */
 function errHandler(err) {
-  if(err.name === 'EarlyHangup' || err.name === 'HangupFailure') {
+  if (isFatal(err)) {
```

Comment:
It looks like you only log the error if it is fatal, but blow up if it is not fatal.